### PR TITLE
Adding a sample for explicitly defining the execution order

### PR DESCRIPTION
**Pull request by SinaChavoshi**
This sample is to address #665

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1668)

### CODE REVIEW

File: samples/basic/execution_order.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      image='library/bash:4.4.23',
+      command=['sh', '-c'],
+      arguments=['echo "$0"', text2])
+
+
+@dsl.pipeline(
+    name='Execution ordere pipeline',
+    description='A pipeline to demonstrate execution order management.'
+)
+
```
Comment (Ark-kun) [inline samples/basic/execution_order.py]: Nit: These blank lines are confusing.

File: samples/basic/execution_order.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+def echo2_op(text2):
+  return dsl.ContainerOp(
+      name='echo2',
+      image='library/bash:4.4.23',
+      command=['sh', '-c'],
+      arguments=['echo "$0"', text2])
+
+
+@dsl.pipeline(
+    name='Execution ordere pipeline',
```
Comment (Ark-kun) [inline samples/basic/execution_order.py]:
```suggestion
    name='Execution order pipeline',
```

File: samples/basic/execution_order.py (hunk 1 above)
Comment (SinaChavoshi) [reply samples/basic/execution_order.py]: Thank you, updated accordingly.

**Pull request merged**